libs/ui/src/hooks/use-search-new-token.hook.ts

```diff
@@ -6,10 +6,12 @@ import { EMPTY_STRING } from '../constants/defaults';
 import { Token, TokenFormType } from '../interfaces/token.interface';
 import { checkTokenOnExist } from '../screens/tokens/utils/compare.util';
 import { createEntity } from '../store/utils/entity.utils';
+import { useAccountTokensAndGasTokenSelector } from '../store/wallet/wallet.selectors';
 
 import { useGetTokenMetadata } from './use-get-token-metadata.hook';
 
-export const useSearchNewToken = (tokens: Token[]) => {
+export const useSearchNewToken = () => {
+  const tokens = useAccountTokensAndGasTokenSelector();
```

Comment:
maybe better to use here `useTokensMetadataSelector`

Reply:
we can't use search by metadata because all metadata is saved with [chainId, tokenAddress] key, and if we will add a token to account 1 then this token will also be available to other accounts, but on other accounts, we didn't add this token

Reply:
we can use this logic jointly use the `useAccountTokensAndGasTokenSelector` for check tokens by current account